### DISCUSSION TITLE
Feature request: digest mode for "watch" emails

### DISCUSSION THREAD

**Discussion by lgritz** [community/community#59457] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When you "watch" a repo, you get email alerts about new PRs, Issues, Discussions -- a separate email for every one, which can be A LOT for an active repo.

You can throttle it back by only getting those alerts for mentions/follows, or only for releases, but then you might miss a lot.

It would be really great if you could get every alert, but get it in a "digest mode" where you would get just one email per day summarizing them all (much like email lists usually have digest modes). This would be very handy for people who want to watch all the activity on a repo so they don't miss something they are interested in, but don't want or need up-to-the minute latency and an endless barrage of distractions by email arrivals that each must be read and usually deleted.

The digest email could be brief, one-liners and links for each PR/Issue/Discussion, maybe it could look something like this:

---

MyProject digest for 2023-06-27

**PRs**

- \#123 Convert code base from C++ to LISP
- \#125 Update the README to add lasagna recipe

**Issues**

- \#124 Feature request: more dinner recipes in README

**Discussions**

- None

**Releases**

- Release 0.9.0

**Comment by rmcdermo** [comment]
It is remarkable that this does not seem to have been solved.  We are about to migrate away from Google Group Discussions to GitHub Discussions, and the number one complaint is that GitHub does not have a digest.  I've been searching forums for solutions.  Am I missing something obvious?  Thanks
